### DISCUSSION TITLE
Calling all touchscreens! I can't Click notes on lazer new update with my touchscreen

### DISCUSSION THREAD

**Discussion by OSUsimp** [ppy/osu#18355] [Q&A]
### Type

Game behaviour

### Bug description

Problem details: Teleporting cursor on ous lazer

Hello, I have a question. Lazer has some problems for me, I play on the surface ( a window laptop ). With this device, I can click the notes without having to drag my hand on the screen and using the keyboard. it seems like I'm having Connection issues with the version now, it works perfectly on stable. this has been uneasy having to play the other way to enjoy Lazer any way to fix this? I have already tried to see if it was my skin or even try playing around with the settings. try to follow the forums, but I still can't get notes even if I tap hard enough! If it is Please comment. if you do have a touch screen do you have the same problems or its just me?

i post videos if i have too

### Version

 Lazer 2022.515.0

### Logs

https://osu.ppy.sh/community/forums/topics/1578112?n=1

**Comment by Joppe27** [comment]
Surface Book 2 here, works just fine on the last version. Make sure you have the "high precision mouse" and "disable mouse buttons during gameplay" settings turned off.

**Reply by peppy** [reply]
For what it's worth, I'm aware of and tracking a fix for this.

**Reply by frenzibyte** [reply]
If it's indeed relevant to "high precision mouse", then we can continue tracking this at https://github.com/ppy/osu-framework/issues/5033

**Reply by peppy** [reply]
That's the one.
